src/cascadia/TerminalSettingsModel/FontConfig.cpp

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "FontConfig.h"
+#include "FontConfig.g.cpp"
+#include "TerminalSettingsSerializationHelpers.h"
+#include "JsonUtils.h"
+
+using namespace Microsoft::Terminal::Settings::Model;
+using namespace winrt::Microsoft::Terminal::Settings::Model::implementation;
+
+static constexpr std::string_view FontInfoKey{ "font" };
+static constexpr std::string_view FontFaceKey{ "fontFace" };
+static constexpr std::string_view FontSizeKey{ "fontSize" };
+static constexpr std::string_view FontWeightKey{ "fontWeight" };

```

Comment:
So we're not going with this design? (remove the "font" prefix from each entry)
```json
"font":{
    "face": "Consolas",
    "size": 12,
    "weight": "light"
}
```

Reply:
_Oops_ you're right, fixed!